content/collections/analytics/en/microscope.md

```diff
@@ -22,8 +22,7 @@ This feature is available to users on **all Amplitude plans**. See the [pricing
 ## Before you begin
 
 * Microscope streams only highlight events selected in the chart's [Events Module](/docs/analytics/charts/build-charts-add-events) that occur during the date range selected.
-* *View Groups* and *Download Groups* are only available to Enterprise customers.
```

Comment:
https://discourse.amplitude.com/t/clarification-on-microscope-docs/15418 you dont need to be enterpise